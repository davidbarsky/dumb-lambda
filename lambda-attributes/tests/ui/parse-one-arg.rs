#![feature(async_await)]

#[lambda_attributes::lambda]
async fn main() {}
