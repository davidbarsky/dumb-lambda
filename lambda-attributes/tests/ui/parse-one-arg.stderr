error: fn main can only take 1 argument
 --> $DIR/parse-one-arg.rs:3:1
  |
3 | #[lambda_attributes::lambda]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^

error[E0601]: `main` function not found in crate `$CRATE`
  |
  = note: consider adding a `main` function to `$DIR/tests/ui/parse-one-arg.rs`

For more information about this error, try `rustc --explain E0601`.
